bayes_kit/ess.py

```diff
@@ -71,7 +23,7 @@ def first_neg_pair_start(chain: VectorType) -> IntType:
     while n + 1 < N:
         if chain[n] + chain[n + 1] < 0:
             return n
-        n = n + 2
+        n += 2
     return N
```

Comment:
Just to make sure this is working as intended:

If given [2, -1, -3, -5, 6, 12], should this function return 2 or 1?

As written it returns 2, since it always counts pairwise, but obviously -1 + -3 is -4 (< 0), and the docstring doesn't indicate that the pairs have to start on an even index.